tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py

```diff
@@ -619,8 +618,30 @@ async def with_airbyte_java_connector(context: ConnectorContext, connector_java_
         .with_directory("builts_artifacts", build_stage.directory("/airbyte"))
         .with_exec(["sh", "-c", "mv builts_artifacts/* ."])
         .with_exec(["rm", "-rf", "builts_artifacts"])
-        # TODO get version from metadata
-        .with_label("io.airbyte.version", connector_version)
-        .with_label("io.airbyte.name", f"airbyte/{application}")
+        .with_label("io.airbyte.version", context.metadata["dockerImageTag"])
+        .with_label("io.airbyte.name", context.metadata["dockerRepository"])
         .with_entrypoint(["/airbyte/base.sh"])
     )
+
+
+def with_airbyte_python_connector(context: ConnectorContext, build_platform: Platform):
+    pip_cache: CacheVolume = context.dagger_client.cache_volume("pip_cache")
+
+    return (
+        context.dagger_client.container(platform=build_platform)
+        .from_("python:3.9-slim")
+        .with_exec(["apt-get", "update"])
```

Comment:
@evantahler I did not thoroughly checked all the images and assumed all the Dockferfile the same (except for java connectors that optionally enable sentry).
Can we go away with this assumption and reconsider it if the acceptance tests are failing?
For this `source-faker` example I think we shall expose a timezone setting in the spec instead of inferring it from the build environment.